stanford/logs-backend.yml

```diff
@@ -4,6 +4,9 @@
   gather_facts: True
   vars:
     LOGSTASH_ELASTICSEARCH_HOST: localhost
+    # take in number of mega bytes, will scale it properly in elasticsearch/tasks 
+    ELASTICSEARCH_HEAP_SIZE_INT: "{{ ansible_memtotal_mb / 2 }}"
```

Comment:
I think you need to explicitly cast this as an integer.
When I installed to the sandbox, the value was `497.0m`

Elasticsearch crashed and refused to start until the decimal part was removed :(